public/app/features/data-connections/hooks/useNavModel.ts

```diff
@@ -9,14 +10,17 @@ import { ROUTE_BASE_ID } from '../constants';
 // (In case we were using `getNavModel()` from app/core/selectors/navModel, then we would need to set
 // the child nav-model-item's ID on the call-site.)
 export const useNavModel = () => {
-  const { pathname } = useLocation();
+  const { pathname: currentPath } = useLocation();
   const navIndex = useSelector((state: StoreState) => state.navIndex);
   const node = navIndex[ROUTE_BASE_ID];
   const main = node;
+  const isDefaultRoute = (item: NavModelItem) =>
+    currentPath === `/${ROUTE_BASE_ID}` && item.id === 'data-connections-datasources';
```

Comment:
Are there user types (e.g. viewer) that shouldn't have access to this route?

Reply:
That's a good point. I am not too sure, but will double check it 👍

Reply:
Instead of tackling it in the current PR I have created a separate issue and will pick it up soon: https://github.com/grafana/grafana/issues/52592